### PR TITLE
remove container and remove cgroup files of container

### PR DESCRIPTION
**Pull request by jhstianwei**
1. remove container then remove cgroup files of container.
2. runc version is aliyun-fc/runc commitid is: 1147685483ef383539d55035b95de038c7c068c5
    which skip destroy cgroup files of container

### CODE REVIEW

File: daemon/delete.go [hunk 1]
```diff
@@ -14,6 +15,52 @@ import (
...
+	"blkio":"/sys/fs/cgroup/blkio/docker/%s",
+	"cpu":"/sys/fs/cgroup/cpu/docker/%s",
+	"cpuacct":"/sys/fs/cgroup/cpuacct/docker/%s",
+	"devices":"/sys/fs/cgroup/devices/docker/%s",
+	"hugetlb":"/sys/fs/cgroup/hugetlb/docker/%s",
+	"net_prio":"/sys/fs/cgroup/net_prio/docker/%s",
+	"perf_event":"/sys/fs/cgroup/perf_event/docker/%s",
+	"pids":"/sys/fs/cgroup/pids/docker/%s"}
+
+// eemovePaths iterates over the provided paths removing them.
```
Comment (rockuw) [inline daemon/delete.go]: eemovePaths => removePaths

File: daemon/delete.go [hunk 2]
```diff
@@ -133,6 +180,17 @@ func (daemon *Daemon) cleanupContainer(container *container.Container, forceRemo
...
 
+	// destroy cgroup files
+	cgroupPaths := make(map[string]string)
+	for s, p := range cgroupFilePaths {
+		cgroupPaths[s] = fmt.Sprintf(p, container.ID)
+	}
+
+	err = removePaths(cgroupPaths)
+	if err != nil {
+		return fmt.Errorf("Fail to Destroy cgroups of container %s, err %s", container.ID, err)
```
Comment (rockuw) [inline daemon/delete.go]: Fail to Destroy => Fail to destroy

**Review by rockuw — APPROVED** [review]

File: daemon/delete.go (hunk 1 above)
Comment (jhstianwei) [reply daemon/delete.go]: done

File: daemon/delete.go (hunk 2 above)
Comment (jhstianwei) [reply daemon/delete.go]: done

**Pull request merged**